DeskKit/DeskKit/View Models/DKContactUsViewModel.m

```diff
@@ -291,9 +291,11 @@ - (NSDictionary *)caseDictionary
 - (NSDictionary *)messageDictionary
 {
     NSString *fromEmail = [self bestFromEmail];
+    NSString *body = self.bodyItem.text.string.dkTrimmedString;
+    
     NSMutableDictionary *dictionary = [@{
                                          DKMessageDirectionKey: @"in",
-                                         DKMessageBodyKey: self.bodyItem.text.string,
```

Comment:
I disagree with the notion that modifying the users content in this case is making the data more accurate. On the contrary, what if the user wants white spaces for a reason, maybe ascii art, who knows. I can certainly imagine other situation where trimming whitespace changes the meaning of what the user intended. Trimming these characters is modifying what the user wanted to send. And as to the question of which is more efficient, we would need measurements of the cost of sending extra characters vs more cpu time on the extra method calls.